Mathlib/Data/Set/Pointwise/SMul.lean

```diff
@@ -1035,6 +1035,22 @@ lemma inv_op_smul_set_distrib (a : α) (s : Set α) : (op a • s)⁻¹ = a⁻¹
 lemma smul_set_disjoint_iff : Disjoint (a • s) (a • t) ↔ Disjoint s t := by
   simp [disjoint_iff, ← smul_set_inter]
 
+example (a : α) (s : Set β) : a • s = (fun b ↦ a • b) '' s := by
+  rw [@image_smul]
```

Comment:
Mathlib typically doesn't have many examples. Are you proposing to keep this in? Or was it a leftover from testing something?

Reply:
No, I'm sorry, it's just a leftover. Let me delete it (and check there are no similar leftovers…).